src/index.html

```diff
@@ -1,13 +1,57 @@
 <!doctype html>
 <html lang="en">
+
   <head>
     <meta charset="UTF-8">
     <meta name="viewport"
           content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <title>Stars</title>
     <link rel="stylesheet" href="./style.css">
   </head>
+
   <body>
-    <h1>Stars</h1>
+    <section class="stars stars--0">
+      <div class="stars__star"></div>
+      <div class="stars__star"></div>
+      <div class="stars__star"></div>
+      <div class="stars__star"></div>
+      <div class="stars__star"></div>
+    </section>
+    <section class="stars stars--1">
+      <div class="stars__star2"></div>
+      <div class="stars__star"></div>
+      <div class="stars__star"></div>
+      <div class="stars__star"></div>
+      <div class="stars__star"></div>
+    </section>
+    <section class="stars stars--2">
+      <div class="stars__star2"></div>
+      <div class="stars__star2"></div>
+      <div class="stars__star"></div>
+      <div class="stars__star"></div>
+      <div class="stars__star"></div>
+    </section>
+    <section class="stars stars--3">
+      <div class="stars__star2"></div>
+      <div class="stars__star2"></div>
+      <div class="stars__star2"></div>
+      <div class="stars__star"></div>
+      <div class="stars__star"></div>
+    </section>
```

Comment:
```suggestion
    <section class="stars stars--2">
      <div class="stars__star2"></div>
      <div class="stars__star2"></div>
      <div class="stars__star"></div>
      <div class="stars__star"></div>
      <div class="stars__star"></div>
    </section>
    
    <section class="stars stars--3">
      <div class="stars__star2"></div>
      <div class="stars__star2"></div>
      <div class="stars__star2"></div>
      <div class="stars__star"></div>
      <div class="stars__star"></div>
    </section>
```

add blank line between multiple sibling blocks of html